styles/_grid.scss

```diff
@@ -0,0 +1,43 @@
+// Define mixins for flexibility
+.row {
```

Comment:
this is for responsive 
you can use col-XX-{{1-12}} to decide the width of the element